projects/organization-management/src/app/pages/user-create/user-create-page.component.ts

```diff
@@ -30,6 +30,12 @@ export class UserCreatePageComponent implements OnInit {
       preferredLanguage: ['en_US', [Validators.required]],
     }),
     roleIDs: this.fb.control([]),
+    budgets: this.fb.group({
+      orderSpentLimit: ['', Validators.pattern('[0-9.]+$')],
+      budget: ['', Validators.pattern('[0-9.]+$')],
+      budgetPeriod: ['weekly'],
```

Comment:
This would allow input like `100.1.1`. How about `Validators.pattern(/^\d+(\.\d{1,2})?$/)`?